juju/juju
# Renaming "default" to "is_default" in the application get output.

## Description of change

Application configuration output was subject to interpretation when displaying default values.
This PR renames "default" to "is_default" to avoid confusion. 

## QA steps

1. Bootstrap
2. Deploy application with some config setting modified
3. run 'juju config' for this application
4. Default values, inherited or equal to charm defined settings, should be described as "is_default: true" instead of "default: true"

## Documentation changes

If the output sample for 'juju config' contains output for default values, the output needs to be updated. 

## Bug reference

https://bugs.launchpad.net/juju/+bug/1709692
